code/build/template/data/config/[=name].ini

```diff
@@ -64,6 +85,8 @@ Pivot           = center
 Scale           = 0.25
 AngularVelocity = 18
 FXList          = FadeIn # ColorCycle
+[+inspector
+OnCreate        = Inspector.RegisterObject ^]

```

Comment:
This can be left in or removed. It's here primarily to give an immediate example in a new `init` project of the inspector interface.